README.md

```diff
@@ -52,20 +55,77 @@ Phase one design features basic functionality and style for multiple features.
+
+   Within under 5 minutes he had an extensive list of choices and wonderful experiences. Liam spent the rest of the day out and about exploring the ancient sites of Italy. Within a few hours he had managed to find delicious street food, pick up a gift for his wife, and feel enriched with a new culture. When his wife came back from an exhausting day of work he had a cab pre planned to take them to a fancy restaurant. Liam was more than enthusiastic to share a passionate story about his day with his wife. The two enjoyed the night so much they planned to come back to Italy for a wedding anniversary.
+
+1. Carl had it with work, he was frustrated and couldn’t find a way to let loose. On a whim he drove off into the countryside of rural America. While driving around was nice, he found little familiarity with any of the towns and wanted a more exciting weekend. Using the Travel Board app Carl was able to find out the town was celebrating a harvest festival. While high strung and tense at first, he begrudgingly pulled up to the fair.
+
+   Within minutes he found what he wanted, a break from his stressful life. He spent the whole day wandering the fair and had numerous novel experiences. By the end of the day he was bone tired, but he was fulfilled. For once his energy was spent on something he wanted to do, as opposed to what his boss wanted.
+
+1. Taylor manages one of the largest renaissance faires in his state. Many numerous and frustrating restrictions have upset his plans to open this Summer. Timing is tight and a sudden shift can cost lots of money in lost revenue. With anxiety overwhelming him and traditional means of advertisement being too slow to react and change he contacted Travel Board.
+
+   For a reasonable fee Taylor was able to advertise to anyone in the area the exciting opportunity to come to his fair. Despite not having everything he wanted, the advertisement was very profitable and brought in many people who were unaware of the faire. With the extra money he made on his advertisement he was able to open again next year and move into a better house with his husband and their new child.
+
+1. Margret was incredibly bored. She had spent her whole highschool life living on cyclical ups and downs with nothing to do on the weekend. In desperation she found a Travel Board. Within under a minute she found a new opportunity to use her brand new driver's license to get out to an unknown music festival.
+
+   With such an exciting prospect she was able to coax her friends out of the house for a fun night. Despite staying up way too late, this experience was meaningful to her. For once she was able to express her own will, and desire to have fun on her own terms. Now Margret is able to express herself socially, and knows many of the exciting upcoming events in her town.
 
```

Comment:
Kind of a lot of these stories, but more content is better at this stage, right? Maybe we can put each of them in their own spoiler?

Reply:
I removed the motivation part right. I think like Joshua. Willian can explain later his idea in a meeting and a summary can be added.